### PR TITLE
Add research data to welcome emails

### PR DESCRIPTION
**Pull request by dju90**
We recently have compiled (or completed with partners) some really strong research backing up the value of learning CS. We would like to add a reference to this research in the emails sent when welcoming a new teacher or parent to our platform after they have either created a new account or linked their email address to their child's account, respectively. Note: the branch name is misleading - it should be 'add-research-data-to-welcome-emails'

## Links

- [spec](https://docs.google.com/document/d/1q3S28_GomttVt-XwrMtRKJdHN_vDjpYpG2QrB44K_oQ/edit)

### CODE REVIEW

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml [hunk 1]
```diff
@@ -34,6 +28,12 @@
 
 %h3
   Stay involved with CS Education
+%p
+  = link_to('Six different studies show', 'https://medium.com/@codeorg/cs-helps-students-outperform-in-school-college-and-workplace-66dd64a69536') + ":"
+  children who study computer science perform better in other subjects,
+  excel at problem-solving,
+  and are 17% more likely to attend college.
+
```
Comment (islemaster) [inline dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]: Note: Trailing space on this line will cause a lint error on staging. (We should figure out how to get Drone to catch this.)

Comment (dju90) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]: I'm sorry, what trailing space are you referring to?

Comment (islemaster) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]: With whitespace rendering enabled in VSCode:

image

I'm referring to the two space characters on line 36.  See line 28 for what this should look like.

Most IDEs have options both to make whitespace visible like this, and to automatically strip trailing whitespace on save so you don't have to think about it.

Comment (dju90) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]: Ah I see. Thanks!

**Review by islemaster — APPROVED** [review]

**Pull request merged**